### PR TITLE
[prometheus-snmp-exporter] Add pod and container security context

### PR DESCRIPTION
**Pull request by tom1299**
Signed-off-by: tom1299 <tom1299@jeeatwork.com>

#### What this PR does / why we need it:
Adds the ability to add pod security and container security context to the deployment by specifying them as values.

#### Which issue this PR fixes
None

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-snmp-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 1.0.1
+version: 1.0.2
```
Comment (monotek) [inline charts/prometheus-snmp-exporter/Chart.yaml:4]: new feature should raise minor version

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-snmp-exporter/Chart.yaml (hunk 1 above)
Comment (tom1299) [reply charts/prometheus-snmp-exporter/Chart.yaml]: Ok, I change to version to `1.1.0`

**Review by monotek — CHANGES_REQUESTED** [review]
please fix linter errors...

**Comment by tom1299** [comment]
I missed that error when I was testing locally. Now all checks should pass.

**Review by monotek — APPROVED** [review]

**Pull request merged**